modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -72,7 +72,7 @@ $ oc delete devworkspaceroutings.controller.devfile.io --all-namespaces --all --
 If this step is not complete, finalizers make it difficult to fully uninstall the Operator easily.
 ====
 +
-. Run the following commands to remove the CRDs:
+. Run the following commands to remove the CRDs used by the operator:
```

Comment:
**Suggestion**: Start with the objective, instead of the action.

To remove the CRDs used by the Operator, run the following commands:

Reply:
@sounix000 Just to clarify, do we need to write "run the following commands" as I think it is understood to the developer/admin that the following are commands only.

Reply:
@rishumehra - You are correct. In that case, we can simply write: "Remove the CRDs used by the Operator.".

This suggestion can be implemented across the entire PR/module.